services/facade/src/repositories/account/account.repository.ts

```diff
@@ -0,0 +1,19 @@
+import {dataSource} from './swagger.datasource';
+/* tslint:disable no-any */
+export class AccountRepository {
+  model: any;
```

Comment:
@b-admike How do we want to get the types into the Facade? I did not want to import them from eg. '../../../account/bla/bla' as we then have a direct coupling. For now I just made the types any, perhaps this example should also be a lerna project in the future, that way the facade can just import the types.

Reply:
+1 for setting up Lerna-based monorepo in this repository. The sooner we can make that transition, the better.